### PR TITLE
Join add tables

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
### In this milestone I implemented the following to my project 

- Created a table named "vets".
- Created a "join table" called "specialization" to handle the many-to-many relationship between "species" and "vets".
- Created a "join table" called "visits" to handle the many-to-many relationship between "animals" and "vets".
- Inserted the data provided in the project requirements into the "vets", "specialization", and "visits" tables.
- Used primary key & foreign key mechanism for joining tables.
- Prepared complex queries that answer analytical questions that were provided in the project requirements.
_Below are screenshots showing the above in the shell cmd 
Screenshot (106)
Screenshot (107)
Screenshot (108)
Screenshot (109)
Screenshot (110)
Screenshot (111)
Screenshot (112)
**screenshots of applied changes as proposed by the previous reviewer  below**
Screenshot (114)
**

### CODE REVIEW

**Review by andersonlebon — COMMENTED** [review]
## Hi @Ngufor-emmanuella 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has merge conflicts and does not contain the required changes from the first review, kindly work on this and resubmit your project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: schema.sql [hunk 1]
```diff
@@ -31,4 +31,41 @@ CREATE TABLE species (
...
+    name VARCHAR(70),
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE specializations (
+    vets_id INT REFERENCES vets (id),
+    species_id INT REFERENCES species (id),
+    PRIMARY KEY (vets_id, species_id)
```
Comment (NduatiKagiri) [inline schema.sql]: - [ ] From the project requirements, a vet can specialize in multiple species, and a species can have multiple vets specialized in it. But in your implementation, a vet can only specialize in one species, and a species can only have one vet specialized in it. Kindly correct that. You can do it by not assigning the Primary Keys.

File: schema.sql [hunk 2]
```diff
@@ -31,4 +31,41 @@ CREATE TABLE species (
...
+);
+
+CREATE TABLE visits (
+    animals_id INT REFERENCES animals(id),
+    vets_id INT REFERENCES vets(id),
+    visit_date date,
+    PRIMARY KEY (animals_id, vets_id, visit_date)
```
Comment (NduatiKagiri) [inline schema.sql]: - [ ] From the project requirements, an animal can visit multiple vets and one vet can be visited by multiple animals, but in your implementation, an animal can only visit one vet and one vet can only be visited by one animal. Kindly correct that. You can do it by not assigning the Primary Keys.

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella,
# Required Changes ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights ✔️
✔️ Gitflow observed
✔️ Good commit message

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please always make sure that when requesting a re-review you submit the PR that the previous reviewers reviewed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Ngufor-emmanuella) [reply schema.sql]: thank you @andersonlebon for giving me feedback . please I have a question. please do you mean I should remove only the primary keys for vets and   animals table ?

**Review by TadesseDev — APPROVED** [review]
### Hi @Ngufor-emmanuella 
### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ST3D


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitFlow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank you

**Pull request merged**